cmd/root_test.go

```diff
@@ -16,7 +16,7 @@ var _ = Describe("Root", func() {
 
 		It("can create commands", func() {
 			commands := cmd.MakeCommands()
-			Expect(len(commands.Commands())).To(Equal(10))
+			Expect(len(commands.Commands())).To(Equal(11))
```

Comment:
Nit: I think we can delete this test.  It's not adding much value to have to increment it each time we add a command.